internal/cmd/commands/dev/dev.go

```diff
@@ -427,11 +436,11 @@ func (c *Command) Run(args []string) int {
 	c.Config.Plugins.ExecutionDir = c.flagPluginExecutionDir
 	c.Config.Worker.AuthStoragePath = c.flagWorkerAuthStorageDir
 	if c.flagIdSuffix != "" {
-		if len(c.flagIdSuffix) != 10 {
+		if len(c.flagIdSuffix) != 10 || len(c.flagSecondaryIdSuffix) != 10 {
```

Comment:
This will not report an error about the primary id suffix when the secondary id suffix is wrong. Do we not want to remove these checks from this if statement?

Reply:
Ugh.. I must've hit undo or something. Thanks for spotting this. Fixed with [0c70f05](https://github.com/hashicorp/boundary/pull/2738/commits/0c70f05ddfe65ab9fdb9d9bfd0b19bf5708b0308)